Formula/gr-osmosdr.rb

```diff
@@ -0,0 +1,19 @@
+class GrOsmosdr < Formula
+  desc "Osmocom Gnu Radio Blocks"
+  homepage "https://osmocom.org/projects/sdr/wiki/GrOsmoSDR"
+  url "https://cgit.osmocom.org/gr-osmosdr/snapshot/gr-osmosdr-0.1.4.tar.xz"
+  sha256 "1945d0d98fd4b600cb082970267ec2041528f13150422419cbd7febe2b622721"
+  # head "git://git.osmocom.org/gr-osmosdr"
+  head "git://github.com/mikemorris/gr-osmosdr.git"
+
+  depends_on "cmake" => :build
+  depends_on "gnuradio"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
```

Comment:
A good test would be a simple source program, compiling, linking and running against the newly built library.